index.html

```diff
@@ -0,0 +1,275 @@
+                
+        <a href="https://twitter.com/josuerushanika"><img src="images/twitter.png" alt="twitter"></a>
+    </li>
+    <li class="social-icon linkedin">
+        <a href="https://www.linkedin.com/in/josue-rushanika-336049b3/"><img src="images/Linkedin.png" alt="linkedin"></a>
+    </li>
+    <li class="social-icon">
+        <a href="#"><img src="images/m.png" alt="m"></a> 
+    </li>
+    <li class="social-icon github">
+        <a href="https://github.com/josuerushanika"><img src="images/github.png" alt="github"></a>
+    </li>
+    <li class="social-icon fingers">
+        <a href="#"><img src="images/fingers.png" alt="fingers"></a>
+    </li>
+ </ul>
+
+ <button class="primary-button resume-button" type="button">
+    Get My Resume
+  </button>
+</div>
+
+<div class="skill-section">
+ <ul>
+    <li class="skill-dropdown-container">
+        <h3 class="intro2">Languages</h3>
+        <button class="arrow-button arrow-down-button" type="button">
+            <img src="images/arrow-right.svg" alt="arrow icon">
+        </button>
+    </li>
+
+    <ul id="languages-dropdown-content" class="skill-dropdown-content">
+        <li>
+            <img src="images/js.svg" alt="javaScript icon" />
+            <span>Javascript</span>
+        </li>
+
+        <li>
+            <img src="images/html.svg" alt="html icon">
+            <span>HTML</span>
+        </li>
+        <li>
+            <img src="images/css.svg" alt="css icon" />
+            <span>CSS</span>
+        </li>
+    </ul>
+
+    <li class="skill-dropdown-container skill-dropdown-border">
+        <h3 class="skill-title">Frameworks</h3>
+        <button class="arrow-button" type="button">
+           <img src="images/arrow-right.svg" alt="arrow icon">
+        </button>
+    </li>
+
+    <li class="skill-dropdown-container skill-dropdown-border">
+        <h3 class="skill-title">Skills</h3>
+        <button class="arrow-button" type="button">
+          <img src="images/arrow-right.svg" alt="arrow icon" />
+        </button>
+     </li>
+  </ul>
+ </div>
+</section> 
+</section>
+
+<section id="contact" class="contact-section">
+  <div class="contact-section-bg">
+    <div class="contact-container">
+      <h4>Contact me</h4>
+      <p>
+        If you have an application you are interested in developing, a
+        feature that you need built or a project that needs coding. I’d love
+        to help with it
+      </p>
+      <form action="https://formspree.io/f/xnqreboq" method="POST">
+        <input type="text" name="name" id="name" placeholder="Name" maxlength="30" required />
+        <input type="email" name="email" id="email" placeholder="Email" required />
+        <textarea name="message" id="message" cols="30" rows="10" placeholder="Write your message here"
+          maxlength="500" required></textarea>
+        <button class="primary-button" type="submit">Get in Touch</button>
+      </form>
+    </div>
+  </div>
+</section>
+
```

Comment:
- You did a great job in your design it looks very nice however it would be better to add the background image in your contact form it will make your design stick to the Figma design as required in the project :+1: 
 Here below your design compare to the Figma design 


## Your design 

![image](https://user-images.githubusercontent.com/95297251/206691043-6fc08716-47c4-4143-8078-dc95bfc69c28.png)

## Figma design

![image](https://user-images.githubusercontent.com/95297251/206690916-1a66ce2b-e4fc-4230-9592-3fe788a7d40d.png)